stages/org.osbuild.zip.meta.json

```diff
@@ -0,0 +1,47 @@
+{
+  "summary": "Assembles the tree into a zip archive named `filename`",
+  "description": [
+    "Buildhost commands used: `zip`"
+  ],
+  "schema_2": {
+    "options": {
+      "additionalProperties": false,
+      "required": [
+        "filename"
+      ],
+      "properties": {
+        "filename": {
+          "description": "Filename for zip archive",
+          "type": "string"
+        },
+        "paths": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "If specified, the archive will only contain the specified paths, in the specified order"
```

Comment:
Might be good to mention that it really wants glob style expressions and that it's just passed to zip.